FlowCryptUI/Nodes/AttachmentNode.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  AttachmentNode.swift
+//  FlowCryptUI
+//
+//  Created by QSD BiH on 16. 4. 2021..
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import AsyncDisplayKit
+
+public struct Attachment {
+    var name, size: String
+
+    public init(
+        name: String,
+        size: Int
+    ) {
+        self.name = name
+        self.size = ByteCountFormatter.string(fromByteCount: Int64(size), countStyle: .file)
+    }
+}
+
+public final class AttachmentsNode: CellNode {
+    public struct Input {
+        let name: String
+        let size: String
+
+        public init(
+            name: String,
+            size: String
+        ) {
+            self.name = name
+            self.size = size
+        }
+    }
+
+    private var attachmentNodes: [AttachmentNode] = []
+    private var onTap: (() -> Void)?
+
+    public init(attachments: [Attachment], onTap: (() -> Void)?) {
+        super.init()
+        self.onTap = onTap
+        attachmentNodes = attachments.map { AttachmentNode(input: AttachmentNode.Input(name: $0.name, size: $0.size),
+                                                           onTap: {
+                                                            self.onTap?()
+                                                           })
+                                                        }
```

Comment:
please change indentation here? hard for understanding 
```
attachmentNodes = attachments.map {
   input: AttachmentNode(
       input:
       onTap:
      )
   )
}
```

Reply:
Can xcode be set to indent automatically on save?

Reply:
We already have a script for that. Will check if it's possible to indent code during build phase. 
Will work on it ASAP to prevent all this "code style" comments